src/tlo/methods/tb.py

```diff
@@ -377,16 +377,16 @@ def __init__(self, name=None, resourcefilepath=None, run_with_checks=False):
             "length of inpatient stay for end-of-life TB patients",
         ),
         # ------------------ scale-up parameters for scenario analysis ------------------ #
-        "do_scaleup": Parameter(
-            Types.BOOL,
-            "argument to determine whether scale-up of program will be implemented"
+        "type_of_scaleup": Parameter(
+            Types.STRING, "argument to determine type scale-up of program which will be implemented, "
+                          "can be 'none', 'target' or 'max'",
         ),
```

Comment:
to help us with checking etc, I think the `type_of_scaleup` should be defined as a categorical:

```suggestion
        "type_of_scaleup": Parameter(
            Types.CATEGORICAL, 
            "argument to determine type scale-up of program which will be implemented",
            categories=['none', 'target', 'max']
        ),
```

Reply:
That seems familiar actually. Okay leave it as string then. Please could you raise an issue so that we fix this for the future?

Reply:
Issue #1442 now raised